ooni/measurements.py

```diff
@@ -139,4 +140,9 @@ def list_measurements(compute_size=False):
             measurements.append(get_measurement(measurement_id, compute_size))
         except:
             log.err("Failed to get metadata for measurement {0}".format(measurement_id))
+
+    if order is not None:
+        reverse = True if order.lower() is 'asc' else False
```

Comment:
IMHO, (any) code should fail aloud on unknown input strings. I'd rather trigger the failure explicitly with sentinel objects or code like that:

```
reverse = {'asc': True, 'desc': False}[order.lower()]
```